agg-src/CMakeLists.txt

```diff
@@ -1,244 +1,23 @@
-PROJECT( antigrain )
+cmake_minimum_required(VERSION 3.13...3.19)
 
-# additional are modified Find routines
-SET ( CMAKE_MODULE_PATH "${antigrain_SOURCE_DIR}/bin" )
+project(antigrain
+    VERSION 2.6
+    DESCRIPTION "subpixel picture resolution algorithms"
+    LANGUAGES CXX C
+)
 
-CMAKE_MINIMUM_REQUIRED( VERSION 2.4.8 )
+option(USE_GPC "use of gpc lib. caution! non-commertial licence only!" OFF)
 
-SET(AGG_MAJOR_VERSION 0 )
-SET(AGG_MINOR_VERSION 1 )
-SET(AGG_BUILD_VERSION 1 )
+# support conan package manager
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)

```

Comment:
Looks like bad practice. We cannot compile without conan, conan should be used transparently